Resources/Changelog/Admin.yml

```diff
@@ -61,3 +61,8 @@ Entries:
       in some cases.', type: Fix}
   id: 9
   time: '2023-10-30T01:28:00.0000000+00:00'
+- author: Vasilis
```

Comment:
from what I understand, the bots automatically write the changes here. You don't have to do that.

Reply:
This is for admin changelog, smugleaf is doing this manually too in her prs. I am fully aware the bots do this

Reply:
Ok 🤔